### ISSUE TITLE
[Feature Request]: Rust's turbofish (`::<>`)

### ISSUE THREAD

**Issue by WillsterJohnson** [rubjo/victor-mono#151]
Rust has a pretty unique piece of syntax for passing type parameters to a function, the turbofish `::<>`.

In practice it's used as follows;

```rs
let foo = has_type_param::</* any type expression here */>();
```

If possible, it would be nice to have a fish-inspired ligature for the first three characters `::<`

My thinking is for the "body" part (`<`), take the `<` of the `<$` ligature and duplicate the curved segment so it's on the top also.
Then for the "tail" part (`::`), simply recycle the `|>` ligature.

I'm an awful artist but here's at least a rough visualisation of what I'm imagining;
![Screenshot from 2023-01-18 19-44-42](https://user-images.githubusercontent.com/85991558/213282365-9a7db996-2ab6-466b-940b-0acc6f2525c6.png)

As far as I'm aware, no other programming language uses `::<` for any syntax, even those with `::` namespace access will never permit `<` at the start of a token name.

**Importance:** nice to have

**Comment by rubjo** [comment]
Sorry for taking so long to respond to this.

Not that the turbofish isnt’t cool, but I’m a bit concerned that the «layout shift» will be quite drastic here, going from :: to |>.

But then again, ligatures can be disabled, so why not. Including them in the next release, right now I have no idea when I will have time to do it :)

**Comment by rubjo** [comment]
Released in 1.5.5.

**Comment by WillsterJohnson** [comment]
awesome! thank you for this - it looks much better than I imagined
